src/rpg_action/communications/slack.clj

```diff
@@ -0,0 +1,41 @@
+(ns rpg-action.communications.slack
+  (:require [clojure.string :as str]
+            [ring.util.response :as ring-response]
+            [clojure.spec.alpha :as s]
+            [rpg-action.models.commands :as commands]
+            [compojure.core :refer :all]
+            [rpg-action.utils :as utils])
+  (:import (clojure.lang PersistentArrayMap)))
+
+(defmulti response type)
+(defmethod response String [resp]
+  (ring-response/response {:text resp}))
+(defmethod response PersistentArrayMap [resp]
+  (ring-response/response resp))
+(defmethod response :default [resp]
+  resp)
+
+(def help-text (utils/long-str "Valid commands: save, list, draw, <action name>"
```

Comment:
I was envisioning /help return what it used to
/help gm returning new gm text (like you've added)
/help dice returning dice notation information (not done at all)
etc

Thoughts?

Reply:
that's probably fine and we can just add a list of things we want to support for help as we think of them